Makefile

```diff
@@ -2,4 +2,4 @@ watch:
 	cargo watch -w miden-lib/asm -x build
 
 test:
-	cargo test --features testing
+	cargo test --release --features concurrent testing
```

Comment:
There was a bug here and in the CI config. This should have been:
```
cargo test --release --features concurrent testing
```

I fixed it in 8ba7faad0a862e12ff941069c4eb7628387de2ad.